src/jobs/consumers/notifications/mailgun-notifications.job.consumer.ts

```diff
@@ -0,0 +1,43 @@
+import { Logger } from '@nestjs/common';
+import { Processor, Process } from '@nestjs/bull';
+import { Job } from 'bull';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Notification } from 'src/modules/notifications/entities/notification.entity';
+import { DeliveryStatus } from 'src/common/constants/notifications';
+import { MailgunService } from 'src/services/email/mailgun/mailgun.service';
+import { MailgunMessageData } from 'mailgun.js';
+import { NotificationsService } from 'src/modules/notifications/notifications.service';
+import { MAILGUN_QUEUE } from 'src/modules/notifications/queues/mailgun.queue';
+
+@Processor(MAILGUN_QUEUE)
+export class MailgunNotificationConsumer {
+  private readonly logger = new Logger(MailgunNotificationConsumer.name);
+
+  constructor(
+    @InjectRepository(Notification)
+    private readonly notificationRepository: Repository<Notification>,
+    private readonly mailgunService: MailgunService,
+    private readonly notificationsService: NotificationsService,
+  ) {}
+
+  @Process()
+  async processMailgunNotificationQueue(job: Job): Promise<void> {
```

Comment:
`Job<number>`

Reply:
This has been changed in the latest commit